ddtrace/contrib/pyramid/patch.py

```diff
@@ -37,3 +39,20 @@ def traced_init(wrapped, instance, args, kwargs):
 
     wrapped(*args, **kwargs)
     trace_pyramid(instance)
+
+def insert_tween_if_needed(settings):
+    if 'pyramid.tweens' not in settings:
+        return
+    tweens_list = settings['pyramid.tweens']
+    # If the list is empty, pyramid does not consider the tweens have been
+    # set explicitely so we will insert the tween with add_tween.
+    if not tweens_list.strip():
+        return
+    # If the our tween is already there, nothing to do
+    if DD_TWEEN_NAME in tweens_list:
+        return
+    idx = tweens_list.find(pyramid.tweens.EXCVIEW)
+    if idx is -1:
+        settings['pyramid.tweens'] = tweens_list + '\n' + DD_TWEEN_NAME
```

Comment:
why the specific placement relative to `EXCVIEW`. leave a comment?

Reply:
@bmermet yes can you explain why we should take this one in consideration? a comment would be great.

Reply:
I'll add a comment. This tween is the default pyramid tween that catches all exception and displays an error view. If our tween is executed after it, we will never see the exception it catches and that should IMO be considered as unhandled.